Neos.Media/Classes/Domain/Model/AssetSource/AssetProxy/AssetProxy.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace Neos\Media\Domain\Model\AssetSource\AssetProxy;
+
+/*
+ * This file is part of the Neos.Media package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+  *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Media\Domain\Model\AssetSource\AssetSource;
+use Psr\Http\Message\UriInterface;
+
+/**
+ * Interface for a stand-in object of remote or already imported assets from an asset source.
+ */
+interface AssetProxy
```

Comment:
Currently our CGL say to add `Interface` to all Interface names. A change to that policy was rebuked a while ago.

Reply:
will fix